src/pseudopeople/configuration/noise_configuration.py

```diff
@@ -107,7 +108,18 @@ def has_noise_type(
         self, dataset_name: str, noise_type: str, column: str | None = None
     ) -> bool:
         dataset_config = self.to_dict()[dataset_name]
-        has_row_noise_type = (
-            "row_noise" in dataset_config and noise_type in dataset_config["row_noise"]
-        )
-        return has_row_noise_type
+        if column:
+            has_noise_type = noise_type in dataset_config["column_noise"][column]
+        else:
```

Comment:
This will trigger if `column == ""` - is that even possible? Maybe you need `if column is not None` above?